fe-nc-news/src/Votes.jsx

```diff
@@ -0,0 +1,27 @@
+import {React, useState, useEffect} from 'react'
+import {patchVotes} from './utils/ApiFunctions'
+
+const Votes = ({article_id, votes}) => {
+    const [userVote, setUserVote] = useState(0)
```

Comment:
We should offer feedback to users if they were to try and vote while offline, we could use axios errors for that 🙂